include/ocpp/v201/utils.hpp

```diff
@@ -61,6 +61,9 @@ std::string generate_token_hash(const IdToken& token);
 /// \return DateTime type timestamp
 ocpp::DateTime align_timestamp(const DateTime timestamp, std::chrono::seconds align_interval);
 
+/// \brief Returns the total Power_Active_Import value from the \p meter_value or 0 if it is not present
+float get_total_power_active_import(const MeterValue& meter_value);
```

Comment:
I would suggest to return an optional here if not found. A power of 0W is a very valid value when idle.